src/devices/sound/ics2115.cpp

```diff
@@ -816,6 +879,52 @@ WRITE8_MEMBER(ics2115_device::write)
 	}
 }
 
+READ16_MEMBER(ics2115_device::read16)
+{
+	uint16_t ret = 0;
+
+	switch(offset) {
+		case 0:
+		case 1:
+			ret = read(space,offset,mem_mask & 0xff);
+			break;
+		case 2:
+		case 3: // TODO : Unknown/Unused at 16 bit accessing?
+			ret = reg_read();
+			break;
+		default:
+#ifdef ICS2115_DEBUG
+			printf("ICS2115: Unhandled memory read at %x\n", offset);
```

Comment:
Please use `logerror` or things from `logmacro.h` rather than `printf` for diagnostics.